chaoslib/litmus/cordon_drain_node.yml

```diff
@@ -37,6 +37,18 @@
         executable: /bin/bash
       register: rescheduled_pod
 
+    # Check that the application pod has rescheduled or not
+    - name: Checking the status of rescheduled pods
+      shell: >
+        kubectl get pod {{ rescheduled_pod.stdout }} -n {{ app_ns }} 
+        --no-headers -o custom-columns=:status.phase
+      args: 
+        executable: /bin/bash
+      register: pod_status
+      until: "pod_status.stdout == 'Running'"
+      delay: 2
+      retries: 90
+
     - name: Wait for application pod reschedule (evict)
```

Comment:
Should these status checks happen before getting a particular pod via shuf (seems later in sequence). Also should it not be status check for all pods within that ns/label rather than a single pod.. The status check can also verify container status IMO.